etl/steps/data/grapher/who/2024-08-06/mortality_database_cancer_most_common.py

```diff
@@ -0,0 +1,23 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset, grapher_checks
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("mortality_database_cancer_most_common")
+    tables = [ds_garden[tab] for tab in sorted(ds_garden.table_names)]
```

Comment:
is there any particular reason why you want `tables` to be sorted by the table names?

If not, there is a simpler alternative to get a list with all tables from the dataset:

```suggestion
    tables = list(ds_garden)
```